### PR TITLE
examples/gcoap_dtls: Increase stack size

### PR DESCRIPTION
**Pull request by maribu**
### Contribution description

As the title says. The stack size was copied over from `tests/pkg_tinydtls_sock_async` cargo-cult style. Also some typos in the README were fixed.

### Testing procedure

Previously on my Nucleo-F767ZI, the CoAP client triggered the MPU stack guard on sending the DTLS hello. With this, the board no longer crashes.

### Issues/PRs references

None

### CODE REVIEW

**Comment by benpicco** [comment]
looks like `Makefile.ci` now needs an update

File: examples/gcoap_dtls/Makefile [hunk 1]
```diff
@@ -94,3 +94,4 @@ ifndef CONFIG_KCONFIG_MODULE_GCOAP
 GCOAP_RESEND_BUFS_MAX ?= 2
 CFLAGS += -DCONFIG_GCOAP_RESEND_BUFS_MAX=$(GCOAP_RESEND_BUFS_MAX)
 endif
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(2*THREAD_STACKSIZE_LARGE\)
```
Comment (benpicco) [inline examples/gcoap_dtls/Makefile:97]: Feels a bit overkill - what is `ps` saying?

**Comment by benpicco** [comment]
I suppose this is still needed, right?

**Comment by maribu** [comment]
It is also needed with `LWIP=1` actually. I wonder if stack usage of the client can be reduced a bit instead. Overflowing a 1.5 KiB stack is a bit surprising.

**Comment by maribu** [comment]
I'm cleaning up my stale branches currently. I think this is actually still needed. Any change to get this in? (Once someone fixes the surprising amount of stack consumption, this is a change that is trivial to revert.)

**Review by benpicco — APPROVED** [review]

**Comment by maribu** [comment]
bors merge

**Comment by maribu** [comment]
bors cancel
bors merge

**Comment by maribu** [comment]
bors cancel
bors merge

let's see if we can get three PRs into the merge train.

**Comment by maribu** [comment]
bors cancel
bors merge

**Comment by maribu** [comment]
rebased and ran the tool to update the `Makefile.ci`

bors retry

**Comment by maribu** [comment]
Thx

**Pull request merged**